lua/telescope/builtin/internal.lua

```diff
@@ -305,8 +305,8 @@ end
 
 internal.vim_options = function(opts)
   -- Load vim options.
-  local vim_opts = loadfile(utils.data_directory() .. path.separator .. 'options' ..
-                            path.separator .. 'options.lua')().options
```

Comment:
Can we do `Path:new({data_dir, options, options.lua}):absolute()` to get the filepath (or similar)